crypto/x509/by_file.c

```diff
@@ -89,22 +87,21 @@ static int by_file_ctrl(X509_LOOKUP *ctx, int cmd,
 int X509_load_cert_file_ex(X509_LOOKUP *ctx, const char *file, int type,
                            OSSL_LIB_CTX *libctx, const char *propq)
 {
-    int ret = 0;
     BIO *in = NULL;
-    int i, count = 0;
+    int count = 0;
     X509 *x = NULL;
 
     in = BIO_new(BIO_s_file());
+    if (in == NULL) {
+        ERR_raise(ERR_LIB_X509, ERR_R_BUF_LIB);
```

Comment:
ERR_R_BIO_LIB

Reply:
So for function BIO_read_filename(), also maybe there should be ERR_R_BIO_LIB code?

Reply:
Yes, sure

Reply:
Fixed